sync/optimistic.md

```diff
@@ -163,6 +163,24 @@ the merge block MUST be treated the same as
 an `INVALIDATED` block (i.e., it and all its descendants are invalidated and
 removed from the block tree).
 
+### How to apply `latestValidHash` when payload status is `INVALID`
+
+Processing the `INVALID` payload status by consensus engine
+depends on the value returned in `latestValidHash` parameter.
+The general approach is as follows:
+1. Consensus engine MUST identify `invalidBlock` as per definition in the table below
+2. `invalidBlock` and all its descendants MUST be transitioned from `NOT_VALIDATED` to `INVALIDATED`
```

Comment:
Perhaps we can add a sentence to the effect that the block with payload `latestValidHash` has to be marked as VALID?

Reply:
Oh but I see you explicitly wanted to only use no valid semantics, this is fine by me

Reply:
Yes, I would keep it conservative as long as we're no sure that ELs support the `VALID` part of semantics correctly.